docs/contracts.rst

```diff
@@ -604,14 +604,14 @@ Like any function, the fallback function can execute complex operations as long
 
     contract Caller {
         function callTest(Test test) public {
-            test.call(abi.encodeWithSignature("nonExistingFunction()"));
+            address(test).call(abi.encodeWithSignature("nonExistingFunction()"));
             // results in test.x becoming == 1.
 
             // The following will not compile, but even
```

Comment:
This is not true anymore. Perhaps use a non-payable regular function?

Reply:
Updated